src/main/java/org/humancellatlas/ingest/project/ProjectService.java

```diff
@@ -175,4 +184,40 @@ private ProjectBag gather(Project project) {
         return new ProjectBag(projects, envelopes);
     }
 
+    public Page<Project> filterProjects(SearchFilter searchFilter, Pageable pageable) {
+        Query query = this.buildProjectsQuery(searchFilter);
+        log.debug("Project Search query: " + query.toString());
+
+        List<Project> projects = mongoTemplate.find(query.with(pageable), Project.class);
+        long count = mongoTemplate.count(query, Project.class);
+        return new PageImpl<>(projects, pageable, count);
+    }
+
+    public Query buildProjectsQuery(SearchFilter searchFilter) {
```

Comment:
This should not be `public` in the `ProjectService`.

Reply:
agreed, but I cant call it in the tests unless it's public.

any suggestions?

Reply:
Yes. It does not belong in this class.

Reply:
This test was made public so it can be called from the tests.
If you are not using it in the tests, it can be made private again.

Reply:
where would you suggest moving it to?

Reply:
It's being called in the ProjectService, so I can't make it private